### ISSUE TITLE
Severe Lag, Not compatible with Chrome OS, blockquotes: [lag] [E-Editable Reblogs][E-Tweaks]

### ISSUE THREAD

**Issue by warpedellipsis** [new-xkit/XKit#321]
Two related things. Old xkit worked with this computer (Acer Chromebook 13), but (1) using more than two extensions or any of the ones that say "this might slow your computer down" now lags me to the point that my mouse won't scroll without a delay. The new one is a CPU hog, but idk why.

(2) The Editable Reblogs extension also screws with other people trying to reblog from me; multi-level block quotes get smashed into one level so that you can't tell who said what. It seems that nobody else has this issue since I know other people who use it and this doesn't happen from their blogs, not even those with my same theme. I'm fairly sure this glitch is also Chrome OS.

![screenshot 2015-08-18 at 21 39 18](https://cloud.githubusercontent.com/assets/13861059/9346992/cde5fc90-45f1-11e5-90bc-38116d6cd65b.png)

**Comment by nightpool** [comment]
I can't think of anything that would be unique to chrome OS that would
cause the issue with editable reblogs. Do you have any other userscripts or
chrome extensions installed?

the 1st problem might be a simple limitation of your hardware. Are you sure
Old XKit with the same extensions was working fine? Can you re-enable it
and disable new xkit to confirm this?

**Comment by dlmarquis** [comment]
I got the second problem once when I attempted to reblog from someone's blog. Their comment wasn't blockquoted, and trying to add it manually made Tumblr refuse to nest blockquotes.

Just tested again---tumblr's fucked with their rich text editor to prevent nested blockquotes. HTML works fine though

**Comment by warpedellipsis** [comment]
I was using old xkit with this machine previously, right before I installed this one. I made sure to delete it from all my accounts on here before putting in the new xkit. I went through all my Chrome extensions before posting this to make sure it wasn't them, but I'm doing it again today to see if the slowness goes away. I know things were fine up through July, it was only recently I got the lag--around the same time I got the new xkit, hence my suspicion. 

I haven't dropped it or anything, and it's brand new (<6mo), so it's not a hardware failure. 

I know the editable reblogs was responsible for making the blockquotes go funny. I removed and tested that, it was definitely that. I had to edit it manually with the HTML to get it to nest right. After that I just deleted the ER from xkit. Sped my machine up and the reblogs quit effing up.

**Comment by dlmarquis** [comment]
I removed and tested it, and blockquotes are still broken for me in the rich text editor. This was tested in a new post, not a reblog, and with Editable Reblogs disabled. What's your browser?

**Comment by dlmarquis** [comment]
Just tested sans XKit -- it's a Tumblr ~~issue~~ "feature".

**Comment by nightpool** [comment]
@dlmarquis I can't reproduce this on my machine? Copying and pasting is
broken but editable reblogs is working fine for me.

**Comment by dlmarquis** [comment]
@nightpool It's not an Editable Reblogs thing. Make a new post and try to nest blockquotes in the rich text editor. That doesn't work at all for me.

**Comment by nightpool** [comment]
@dlmarquius I don't think the rich text editor tools ever supported that.
at least not in the past ~6 months I've been using them

**Comment by dlmarquis** [comment]
@nightpool I wouldnt know, but I now they won't stay put if you move from HTML to rich text, etc. That's new.

**Comment by nightpool** [comment]
@dlmarquis today's update, after going from rich-text -> HTML -> rich text:
![image](https://cloud.githubusercontent.com/assets/233815/9370066/22db09e2-4693-11e5-8c46-3b47c3c9dabe.png)
the reported bug looks more like this:
![image](https://cloud.githubusercontent.com/assets/233815/9370095/60ccaecc-4693-11e5-9fc0-93ae35c24b4b.png)
(this is a mockup—I've seen it happen elsewhere but I've never been able to reproduce it)

**Comment by dlmarquis** [comment]
@nightpool Okay, I can't reproduce it. I'm getting all sorts of fun js errors due to the shitty hotel internet. It might be a speed/interruption issue. Which we know Editable Reblogs can make worse.

**Comment by warpedellipsis** [comment]
You have to go into the HTML editor to get nesting blockquotes. I thought that's how it's always been? I never could get the RichText to nest properly, ever. I'm in Chrome. The screencap I posted is from the  bug in ER in xkit, it's taking all the nested blockquotes and mashing them into the furthest to right one, as pictured in my OP upthread.

And for removing all other bits from my machine, I'm going at normal speed, no lag now. I don't have new xkit in either. Adding in old xkit---I get the conflicts warning, but it lists nothing. I'm guessing that's due to it being old and conflicting with new tumblr "features". Had it installed for over an hour now, and no lagging. GOing to try adding in my other chrome extension...no problems. It was definitely new xkit causing my issues. That checks with my troubleshooting from before, and from when I'd close tumblr and have things speed up.

So unless Chrome OS decided to update and broke new-xkit somehow, there's something wrong with new-xkit that makes it hog my cpu. ER and tweaks hog stuff a lot too.

**Comment by nightpool** [comment]
@warpedellipsis other people have been having the blockquotes problem you
have so I'm guessing its safe to say its not associated with chrome os

**Comment by warpedellipsis** [comment]
Good news is that tumblr has native reblog-from-yourself now, so that may have been what causes some conflicts and problems. Delete that xkit bit and see if it fixes things.

**Comment by warpedellipsis** [comment]
I've swapped out the old xkit to the new one. No lag immediately. However, I only put in the same few extensions i had in oldxkit. I have installed: xinbox, tweaks, activity+, oneclick reply.

It lagged when I enabled the options in Xinbox for: make the answer box taller when I"m typing a reply to an ask; Automatically expand fan mail without having to click read more. 
Other stuff I enabled, for checking: 

(xinbox) all of tagging while publishing except custom tag; show reply button on private answers; enable inbox search.
(oneclickreply) enable in-dash reply; enable replying to multiple notifications; all tagging options except custom tags. 
(activity+) all except hiding graphs.
(tweaks) wrap tags; slim blockquotes; slim quote post text; 

Added outbox, all enabled. Still okay. I've been streaming music and still okay. I've seen elsewhere that wrapped tags are busted but mine are working fine.
